<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlavnoOkno</class>
 <widget class="QMainWindow" name="GlavnoOkno">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>682</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>BenRačun: glavno okno</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>srcek.svg</normaloff>srcek.svg</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbl_pozdrav">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_pozicija">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_datum">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_home">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Domov</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_uporabnik">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Pomen se bo pokazal s časom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_pozicija">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 170, 255);
background-color: rgb(255, 170, 255);
border:0;</string>
        </property>
        <property name="text">
         <string>Pomen se bo pokazal s časom</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>796</width>
         <height>582</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRa_unovodstvo">
    <property name="title">
     <string>Računovodstvo</string>
    </property>
    <widget class="QMenu" name="menuIzdani_ra_uni">
     <property name="title">
      <string>Izdani računi</string>
     </property>
     <addaction name="actionPredra_uni"/>
     <addaction name="actionPredpla_ilni_ra_uni"/>
     <addaction name="actionRa_uni"/>
     <addaction name="actionStornirani_ra_uni"/>
    </widget>
    <addaction name="actionPotni_nalogi"/>
    <addaction name="actionPrejeti_ra_uni"/>
    <addaction name="menuIzdani_ra_uni"/>
   </widget>
   <widget class="QMenu" name="menuIzpisi">
    <property name="title">
     <string>Pregledi</string>
    </property>
    <addaction name="actionPrihodek"/>
    <addaction name="actionIzdatek"/>
    <addaction name="actionPoslovanje"/>
   </widget>
   <widget class="QMenu" name="menuPomo">
    <property name="title">
     <string>Pomoč</string>
    </property>
    <addaction name="actionVizitka"/>
   </widget>
   <widget class="QMenu" name="menuDelo">
    <property name="title">
     <string>Delo</string>
    </property>
    <addaction name="actionStranke"/>
    <addaction name="separator"/>
    <addaction name="actionProjekti"/>
    <addaction name="separator"/>
    <addaction name="action_asovnice"/>
    <addaction name="separator"/>
    <addaction name="actionKuponi"/>
   </widget>
   <widget class="QMenu" name="menuNastavitve">
    <property name="title">
     <string>Podatki</string>
    </property>
    <addaction name="actionPodjetje"/>
    <addaction name="actionUporabniki"/>
    <addaction name="actionAvtomobili"/>
    <addaction name="separator"/>
    <addaction name="actionDopusti_bolni_ke"/>
    <addaction name="separator"/>
    <addaction name="actionNastavitve"/>
   </widget>
   <widget class="QMenu" name="menu_ifranti">
    <property name="title">
     <string>Šifranti</string>
    </property>
    <addaction name="actionStoritve"/>
    <addaction name="actionKategorije"/>
    <addaction name="actionPodkategorije"/>
    <addaction name="separator"/>
    <addaction name="actionOpombe_pri_ra_unih"/>
    <addaction name="separator"/>
    <addaction name="actionVrednost_DDV"/>
    <addaction name="separator"/>
    <addaction name="actionKilometrina"/>
    <addaction name="actionDnevnice"/>
    <addaction name="actionCena_malice"/>
    <addaction name="actionDelavniki"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuDelo"/>
   <addaction name="menuRa_unovodstvo"/>
   <addaction name="menu_ifranti"/>
   <addaction name="menuIzpisi"/>
   <addaction name="menuNastavitve"/>
   <addaction name="menuPomo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVizitka">
   <property name="text">
    <string>Vizitka</string>
   </property>
  </action>
  <action name="actionStranke">
   <property name="text">
    <string>Stranke</string>
   </property>
  </action>
  <action name="actionProjekti">
   <property name="text">
    <string>Projekti</string>
   </property>
  </action>
  <action name="actionPotni_nalogi">
   <property name="text">
    <string>Potni nalogi</string>
   </property>
  </action>
  <action name="actionPrejeti_ra_uni">
   <property name="text">
    <string>Prejeti računi</string>
   </property>
  </action>
  <action name="actionPrihodek">
   <property name="text">
    <string>Prihodek</string>
   </property>
  </action>
  <action name="actionIzdatek">
   <property name="text">
    <string>Izdatek</string>
   </property>
  </action>
  <action name="actionPoslovanje">
   <property name="text">
    <string>Poslovanje</string>
   </property>
  </action>
  <action name="actionUporabniki">
   <property name="text">
    <string>Zaposleni</string>
   </property>
  </action>
  <action name="actionPodjetje">
   <property name="text">
    <string>Podjetje</string>
   </property>
  </action>
  <action name="actionKuponi">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Kuponi</string>
   </property>
  </action>
  <action name="actionPopusti">
   <property name="text">
    <string>Popusti</string>
   </property>
  </action>
  <action name="action_asovnice">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Časovnice</string>
   </property>
  </action>
  <action name="actionOpombe_pri_ra_unih">
   <property name="text">
    <string>Opombe pri računih</string>
   </property>
  </action>
  <action name="actionKilometrina">
   <property name="text">
    <string>Kilometrina</string>
   </property>
  </action>
  <action name="actionDnevnice">
   <property name="text">
    <string>Dnevnice</string>
   </property>
  </action>
  <action name="actionOsnovni_Pogled">
   <property name="text">
    <string>Osnovni Pogled</string>
   </property>
  </action>
  <action name="actionOsnovni_Pogled_2">
   <property name="text">
    <string>Osnovni Pogled</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionNastavitve">
   <property name="text">
    <string>Nastavitve</string>
   </property>
  </action>
  <action name="actionAvtomobili">
   <property name="text">
    <string>Avtomobili</string>
   </property>
  </action>
  <action name="actionPredra_uni">
   <property name="text">
    <string>Predračuni</string>
   </property>
  </action>
  <action name="actionPredpla_ilni_ra_uni">
   <property name="text">
    <string>Predplačilni računi</string>
   </property>
  </action>
  <action name="actionRa_uni">
   <property name="text">
    <string>Računi</string>
   </property>
  </action>
  <action name="actionCena_malice">
   <property name="text">
    <string>Cena malice</string>
   </property>
  </action>
  <action name="actionDopusti_bolni_ke">
   <property name="text">
    <string>Dopusti, bolniške</string>
   </property>
  </action>
  <action name="actionDelavniki">
   <property name="text">
    <string>Delavniki</string>
   </property>
  </action>
  <action name="actionVrednost_DDV">
   <property name="text">
    <string>Vrednost DDV</string>
   </property>
  </action>
  <action name="actionStornirani_ra_uni">
   <property name="text">
    <string>Stornirani računi</string>
   </property>
  </action>
  <action name="actionStoritve">
   <property name="text">
    <string>Storitve</string>
   </property>
  </action>
  <action name="actionKategorije">
   <property name="text">
    <string>Kategorije</string>
   </property>
  </action>
  <action name="actionPodkategorije">
   <property name="text">
    <string>Podkategorije</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
